asm1/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment 1</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!--TOP-->
+    <div class="Top">
+        <p>Palmtree is always with you to explore the world</p>
+        <img src="images\logotravel.PNG">
+        <div class="placeholder">
+            <form>
+                <input type="text" placeholder="Search in website">
+                <button type="submit">Search</button>
+            </form>
+        </div>
+    </div>
+
+    <!--Menu-->
+    <div class="Menu">
+        <ul>
+            <li>Home</li>
+            <li>Tours</li>
+            <li>News</li>
+            <li>Services</li>
+            <li>Events</li>
+        </ul>
+    </div>
+
+    <!--Body 1-->
+    <div class="Body_1">
+        <img src="images\Bannerquangcao.jpg">
+        <div>
+            <h3>Unforgettable experience about Ha Giang</h3>
+            <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid vel et cumque aut reprehenderit impedit expedita odio, 
+                eum non quasi distinctio id, sapiente voluptatem neque? Soluta necessitatibus aliquid asperiores quos!</p>
+            <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid vel et cumque aut reprehenderit impedit expedita odio, 
+                eum non quasi distinctio id, sapiente voluptatem neque? Soluta necessitatibus aliquid asperiores quos!</p>
+        </div>
+        <div class="Registration">
+            <h3>Tour Registration</h3>
+            <h4>Full Name</h4>
+            <input type="text" placeholder="Enter your full name">
+            <h4>Email address</h4>
+            <input type="text" placeholder="Enter your email">
+            <h4>Phone</h4>
+            <input type="text" placeholder="Enter your phone number">
+            <button type="submit">Send</button>
```

Comment:
Use <label> instead of <h4>

```suggestion
    <label for="fullname">Full Name: </h4>
    <input type="text" name="fullname" placeholder="Enter your full name">
    <label for="email">Email address: </h4>
    <input type="text" name="email" placeholder="Enter your email">
    <label for="phone">Phone: </h4>
    <input type="text" name="phone" placeholder="Enter your phone number">
```